lib/mongoid/association/macros.rb

```diff
@@ -12,9 +12,11 @@ module Macros
         class_attribute :embedded, instance_reader: false
         class_attribute :embedded_relations
         class_attribute :relations
+        class_attribute :aliased_relations
         self.embedded = false
         self.embedded_relations = BSON::Document.new
         self.relations = BSON::Document.new
+        self.aliased_relations = {}
```

Comment:
If possible I would like new code to use the term "association" instead of "relation". I understand that the additions match the existing style but my preference nonetheless is to use correct terminology in new code. Do you have any objections to using "association" in new code? To clarify, none of the existing code should be changed at this time for BC reasons.

Reply:
No objection, `aliased_associations` is fine.